testdata/local/admin.json

```diff
@@ -3,6 +3,6 @@
 { "_id" : "global", "uiv2_url": "http://localhost:3000", "api_url" : "http://localhost:9090", "configdir" : "../config", "domain_name" : "localhost" , "keys": {"fake_ssh_key": "/path/to/key"}, "banner" : "This is an important notification","banner_theme" : "announcement", "github_orgs": ["evergreen-ci"] }
 { "_id" : "service_flags", "github_status_api_disabled" : true, "alerts_disabled" : true, "repotracker_disabled" : true, "scheduler_disabled" : true, "check_blocked_tasks_disabled": true, "github_pr_testing_disabled" : true, "repotracker_push_event_disabled" : true, "cli_updates_disabled" : true, "task_dispatch_disabled" : true, "s3_binary_downloads_disabled": true, "monitor_disabled" : true, "notifications_disabled" : true, "taskrunner_disabled" : true, "background_stats_disabled" : true, "event_processing_disabled" : true, "webhook_notifications_disabled" : true, "jira_notifications_disabled" : true, "slack_notifications_disabled" : true, "email_notifications_disabled" : true, "task_logging_disabled" : true, "cache_stats_job_disabled" : true, "agent_start_disabled" : true, "host_init_disabled" : true, "pod_init_disabled": true, "s3_binary_downloads_disabled": true, "commit_queue_disabled" : true, "cache_stats_endpoint_disabled" : true, "host_allocator_disabled" : true, "pod_allocator_disabled": true, "task_reliability_disabled" : true, "background_reauth_disabled": true, "background_cleanup_disabled": true, "cloud_cleanup_disabled": true, "slack_app_disabled": false, "container_configurations_disabled": false, "unrecognized_pod_cleanup_disabled": true }
 { "_id": "amboy", "name": "evg.service", "database": "amboy_local" }
-{ "_id" : "providers", "aws": {"pod": {"ecs": {"client_type": "mock"}, "secrets_manager": {"client_type": "mock"}}}}
-{ "_id": "spawnhost", "unexpirable_hosts_per_user": 2, "unexpirable_volumes_per_user": 1, "spawn_hosts_per_user": 6 }
+{ "_id" : "providers", "aws": {"pod": {"ecs": {"client_type": "mock"}, "secrets_manager": {"client_type": "mock"}}, "allowed_regions": ["rofl-east"]}}
```

Comment:
Can we use us-east-1 instead? this is the default region so when we eventually implement the distro page and stuff we may need to have it